front/src/common/Map/Map.scss

```
@@ -125,6 +136,20 @@
   }
 }
 
+.compass-container {
+  position: relative;
+  width: 24px;
+  height: 24px;
+}
+
+.compass-needle,
+.compass-cardinal {
+  position: absolute;
+  // top: 50%;
+  left: 50%;
```

Better centering in the circle button with that

```suggestion
  left: 60%;
```